src/ui/components/BottomSheet/BottomSheet.tsx

```diff
@@ -61,9 +61,9 @@ const BottomSheet: React.FC<BottomSheetProps> = ({
   height = 400,
   hasOverlay = false,
   isVisible = false,
-  close,
-  children,
-  style,
+  close = () => {},
+  children = null,
+  style = {},
```

Comment:
What's the reason behind defining defaults for optional props here? I'm not saying it's wrong, would just like to understand your philosophy here.

Reply:
Sure! The thing is that it's a good practice to have default values **when the _prop_ isn't required,** so we can prevent unexpected behaviors. Does that make sense for you?

There's also a [eslint rule](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/require-default-props.md) for that :)